CHANGELOG.md

```diff
@@ -16,7 +16,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Performance enhancements
-* `skbio.tree.nj` wall-clock time was decreased about 66% by vectorizing `skbio.tree._nj._compute_q`. ([#1512](https://github.com/biocore/scikit-bio/pull/1512))
+* `skbio.tree.nj` wall-clock time was decreased about 99% by vectorizing `skbio.tree._nj._compute_q` and avoiding unnecessary recalculations. ([#1512](https://github.com/biocore/scikit-bio/pull/1512), [#1513](https://github.com/biocore/scikit-bio/pull/1513))
```

Comment:
is this 99% still accurate given the `.data` change that was made? Would it be possible to note the size distance matrix that this performance improvement is over?